src/routing.rs

```diff
@@ -214,7 +199,12 @@ pub fn setup_popstate_listener<Ms>(
             if let Some(routing_msg) = routes(url) {
                 update(routing_msg);
             }
-        };
+        } else {
+            // `ev.state()` will be a null JsValue for the landing page.
+            if let Some(routing_msg) = routes(initial_url()) {
```

Comment:
Do we want to rename `initial_url` to `current_url`? I think `initial_url` doesn't make sense with this context and it gets current url under the hood.